web/impact/impact/permissions.py

```diff
@@ -89,14 +93,19 @@ def action_matches(self, action, request):
     def convert_string_to_bool(self, text_value):
         try:
             boolean_value = literal_eval(text_value.title())
-        except:
+        except:  # pragma: no cover
+            # Regarding coverage, it's not clear that we need this code
+            # any more.  See has_object_permissions.
```

Comment:
Removing it will take more thought than I wanted to put into this ticket and it is outside the scope.  It's not just a matter of deleting the class since it is still used for somethings according to Shankar.